tf2onnx/onnx_opset/math.py

```diff
@@ -555,7 +555,7 @@ def version_12(cls, ctx, node, **kwargs):
         dtypes = node.output_dtypes
         ctx.remove_node(node.name)
         ctx.make_node("Inverse", inputs=node.input, outputs=node.output, name=node.name,
-                      shapes=shapes, dtypes=dtypes)
+                      domain=constants.MICROSOFT_DOMAIN, shapes=shapes, dtypes=dtypes)
```

Comment:
Inverse is in the onnx domain, no?
https://github.com/onnx/onnx/blob/master/docs/Operators.md#Inverse

Reply:
No, it is a contrib op.
https://github.com/microsoft/onnxruntime/pull/3485

Reply:
the onnx spec says Inverse is in onnx namespace - we follow that spec. It is correct to put it here regardless if onnxruntime supports it or not since they spec says so.

If onnxruntime implements this in ms domain and you want to support it there, you need to add a  handler here: tensorflow-onnx/tf2onnx/custom_opsets/ms.py.

Reply:
moved to ms.py.